### DISCUSSION TITLE
Where to download more .ts files

### DISCUSSION THREAD

**Discussion by mja-ableton** [acids-ircam/RAVE#83] [Q&A]
I'm a musician just trying this out and not into training yet (although I would love a step by step tutorial for laymen on that). I'd like to download more models. is there a repository of .ts files shared by the community?

**Comment by branlsnyder** [comment]
I believe there is currently a .ts file directory as part of the RAVE VST standalone plugin, where people can submit their own trained models for others to use. Last time I checked, it was just getting started.

**Comment by JLenzy** [comment]
Just commenting to support this as well, it would be great to have a repository (outside the VST API) where we can easily download .ts files to play with.
If there is such a directory I'd be happy to upload a couple of models.

**Reply by jreus** [reply]
+1 I'd also be happy to contribute to such a resource.

**Comment by anaisborie** [comment]
Hello everyone 
I’m really new to all of this, but I succeeded at using the nn~ prebuild for max map on Mac, one thing I can’t understand although, is that I couldn’t find any TS file anywhere in this GitHub, not even in the vst directory. If anyone could send me send some of this TS files, especially those containing vocal sample like in the YouTube video of rave, it would be much appreciated !
Thank you so much !
Anais

**Reply by anaisborie** [reply]
Ok !
So I find the way to make it works in Max Msp on my Mac... well if you open the help folder, a windows in max appears, you press the button, and then your files are downloaded at the right place... as I said I'm new at it, many discoveries at once... 
Hope that could help someone else !

**Comment by klangmensch** [comment]
Sorry, I'm a total noob. I got Rave nn~ running inside max msp with the wheel.ts model. Can someone tell me where to find darbouka.ts from the tutorial video and maybe some additional models? Much appreciated!

**Comment by AffeZwei** [comment]
According to this YouTube video, https://www.youtube.com/watch?v=BYRW8k3n1cE
 there aren't other published models. From the comments: "Unfortunately the pretrained models were provided to us during the NASH Hackathon but the creators of RAVE do not want to distribute them publicly at the moment. They may provide some pre-trained models if you contact them directly. You can also join the NASH discord group to be part of the community on Neural Audio Synthesis!"

**Reply by AffeZwei** [reply]
Does anyone know if this NASH group on Discord is still active? or accessible publicly?

**Comment by caillonantoine** [comment]
You'll find some .ts files [here](https://caillonantoine.github.io/NIME_workshop/) :)

**Reply by AffeZwei** [reply]
Thanks!

**Reply by jeremy-keenan** [reply]
> You'll find some .ts files [here](https://caillonantoine.github.io/NIME_workshop/) :)

Thank you Antoine. Can this link be put on the main RAVE github page? I always forget where to find them.

**Comment by anaisborie** [comment]
Oalalala ! Great, thank you :)
